libs/s3/s3_versioning_common_test_lib.py

```diff
@@ -227,7 +227,7 @@ def check_list_objects(s3_test_obj: S3TestLib, bucket_name: str,
                               "List Objects response does not contain expected object names")
 
 
-def check_get_head_object_version(s3_ver_test_obj: S3VersioningTestLib,
+def check_get_head_object_version(s3_test_obj: S3TestLib, s3_ver_test_obj: S3VersioningTestLib,
```

Comment:
Please check if new argument description are added in docstring

Reply:
Updated the docstring